### PR TITLE
Update RequiredSystemDataInNewApiVersions.ts

### PR DESCRIPTION
**Pull request by rohit-bh**
Convert linter rule from warning to error as per RPC027.

### CODE REVIEW

File: src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const RequiredSystemDataInNewApiVersions: string = "RequiredSystemDataInN
 rules.push({
   id: "R4009",
```
Comment (sanjaiganesh) [inline src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts:11]: Under **How to fix**, suggest to use common resource definition (by ARM team) from the following  that has systemdata 
specification\common-types\resource-management\v1\types.json

Comment (rohit-bh) [reply src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts:11]: Do you mean "How to fix" under docs @ https://github.com/Azure/azure-openapi-validator/tree/217995a157bcc01a1081efb2bc56b7465daa1585/docs?
There doesn't seem to be a file for this rule. Should I create one?

Comment (sanjaiganesh) [reply src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts:11]: sorry for the confusion. actually here [https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r3025](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r3025)

---
In reply to: [499902971](https://github.com/Azure/azure-openapi-validator/pull/225#discussion_r499902971) [](ancestors = 499902971)

Comment (jianyexi) [reply src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts:11]: Thanks, good feedback, I will improve the how-to-fix
@josefree  About changing this rule to error, do you have any  suggestion ?

**Comment by jianyexi** [comment]
@rohit-bh  could you fix  the CI error ?

**Comment by rohit-bh** [comment]
> @rohit-bh could you fix the CI error ?

Could you help me understand how to fix this? Clicking on https://dev.azure.com/azure-sdk/public/_build/results?buildId=553011 takes me to a 'Build not found.' page. I don't see any tests failing either.

**Comment by jianyexi** [comment]
> > @rohit-bh could you fix the CI error ?
> 
> @jianyexi, Could you help me understand how to fix this? Clicking on https://dev.azure.com/azure-sdk/public/_build/results?buildId=553011 takes me to a 'Build not found.' page. I don't see any tests failing either.
I have re-ran the CI, you can download the 'snapshot.patch'  in the CI details page  , and git apply to the repo ,commit & sync your branch

**Comment by jianyexi** [comment]
Close as already update it in https://github.com/Azure/azure-openapi-validator/pull/227

**Pull request closed without merging**